src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -131,6 +131,10 @@ public bool AcceptUnbind()
         RemoveCellFromColony(target);
 
         stage.HUD.HintText = string.Empty;
+
+        // In case this is the last colony member being unbonded, return the player's microbe state to normal
+        stage.Player.State = Microbe.MicrobeState.Normal;
```

Comment:
Shouldn't this line only run when the last member was removed?
But then shouldn't the logic really go into the microbe class / colony?
@FuJa0815

Reply:
The AcceptUnbind method is for removing microbes by clicking on them after pressing Ctrl+U. Therefore this logic is wrong here, because the player might want to remove multiple members at once before exiting the unbind mode. I'm on my phone right now, therefore I can't really look into the code right now, but there has to be a better place.

Reply:
As of the latest version, the player automatically exits the unbind mode after clicking a cell. What's weird was that it doesn't seem to do that if you're unbinding the only one bound cell, so this logic essentially forces the player's microbe state reset.

Reply:
So there is something here to fix anyway, I'm still pretty busy with other stuff, so if you two could figure out a good fix / change regarding this that'd be great.